[Swift in WebKit] Work towards modularizing WebCore private headers (Part 11)

rr-codes opened the pull request:
#### 89c467dabfe688409c9cf46482e7ab1506423c58
<pre>
[Swift in WebKit] Work towards modularizing WebCore private headers (Part 11)
<a href="https://bugs.webkit.org/show_bug.cgi?id=300484">https://bugs.webkit.org/show_bug.cgi?id=300484</a>
<a href="https://rdar.apple.com/162339896">rdar://162339896</a>

Reviewed by Mike Wyrzykowski.

Continue changing more incorrectly-quoted includes to angle includes.

* Source/WebCore/Modules/Model/InternalAPI/DDMeshDescriptor.h:
* Source/WebCore/Modules/Model/InternalAPI/DDMeshPart.h:
* Source/WebCore/Modules/Model/InternalAPI/DDUpdateMeshDescriptor.h:
* Source/WebCore/Modules/WebGPU/GPU.h:
* Source/WebCore/Modules/applepay/PaymentCoordinator.h:
* Source/WebCore/Modules/webtransport/WebTransportConnectionStats.h:
* Source/WebCore/Modules/webtransport/WebTransportHash.h:
* Source/WebCore/Modules/webtransport/WebTransportOptions.h:
* Source/WebCore/Modules/webtransport/WebTransportSendStreamSink.h:
* Source/WebCore/Modules/webtransport/WorkerWebTransportSession.h:
* Source/WebCore/accessibility/AXUtilities.h:
* Source/WebCore/dom/DocumentPage.h:
* Source/WebCore/dom/DocumentResourceLoader.h:
* Source/WebCore/dom/LoadableSpeculationRules.h:
* Source/WebCore/layout/formattingContexts/grid/GridFormattingContext.h:
* Source/WebCore/layout/formattingContexts/grid/GridItemRect.h:
* Source/WebCore/layout/formattingContexts/grid/GridLayout.h:
* Source/WebCore/layout/formattingContexts/grid/ImplicitGrid.h:
* Source/WebCore/layout/formattingContexts/grid/TrackSizingAlgorithm.h:
* Source/WebCore/layout/formattingContexts/grid/TrackSizingFunctions.h:
* Source/WebCore/layout/formattingContexts/grid/UnplacedGridItem.h:
* Source/WebCore/layout/formattingContexts/grid/UsedTrackSizes.h:
* Source/WebCore/loader/DocumentPrefetcher.h:
* Source/WebCore/loader/cache/CachedResourceHandle.h:
* Source/WebCore/platform/graphics/AudioVideoRenderer.h:
* Source/WebCore/platform/graphics/avfoundation/AudioVideoRendererAVFObjC.h:
* Source/WebCore/style/values/backgrounds/StyleBackgroundLayer.h:
* Source/WebCore/style/values/backgrounds/StyleBackgroundSize.h:
* Source/WebCore/style/values/backgrounds/StyleFillLayers.h:
* Source/WebCore/style/values/backgrounds/StyleRepeatStyle.h:
* Source/WebCore/style/values/masking/StyleMaskLayer.h:

Canonical link: <a href="https://commits.webkit.org/301320@main">https://commits.webkit.org/301320@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/ac1f433ea2806aab9ec30f6b458775873cfbaa09

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/125609 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/45277 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/36027 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/132471 "Built successfully") | [  ~~🛠 win~~](https://ews-build.webkit.org/#/builders/59/builds/77498 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | ⏳ 🛠 ios-apple 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/127480 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/45964 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/53831 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/95688 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/59/builds/77498 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/8fc1079b-004f-48a2-979c-907509455f45) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/128557 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/36739 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/112330 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/76184 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/3dbe05fa-438c-47fe-a9ba-15b9561150a0) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/35641 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/30511 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/75944 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/106517 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/30729 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/135143 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/52408 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/40167 "Passed tests") | [❌ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/104160 "Failed to checkout and rebase branch from PR 52101") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/52850 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/108541 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/103890 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/26450 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/49246 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/27557 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/49626 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/52299 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/58096 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/51654 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/55010 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/53346 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

mwyrzykowski reviewed (approved):
I would suggest ignoring the style checker errors because re-ordering the includes could introduce a breaking change, how you have it is purely mechanical afaik so seems like close to zero risk

rr-codes committed:
[Swift in WebKit] Work towards modularizing WebCore private headers (Part 11)
https://bugs.webkit.org/show_bug.cgi?id=300484
rdar://162339896

Reviewed by Mike Wyrzykowski.

Continue changing more incorrectly-quoted includes to angle includes.

* Source/WebCore/Modules/Model/InternalAPI/DDMeshDescriptor.h:
* Source/WebCore/Modules/Model/InternalAPI/DDMeshPart.h:
* Source/WebCore/Modules/Model/InternalAPI/DDUpdateMeshDescriptor.h:
* Source/WebCore/Modules/WebGPU/GPU.h:
* Source/WebCore/Modules/applepay/PaymentCoordinator.h:
* Source/WebCore/Modules/webtransport/WebTransportConnectionStats.h:
* Source/WebCore/Modules/webtransport/WebTransportHash.h:
* Source/WebCore/Modules/webtransport/WebTransportOptions.h:
* Source/WebCore/Modules/webtransport/WebTransportSendStreamSink.h:
* Source/WebCore/Modules/webtransport/WorkerWebTransportSession.h:
* Source/WebCore/accessibility/AXUtilities.h:
* Source/WebCore/dom/DocumentPage.h:
* Source/WebCore/dom/DocumentResourceLoader.h:
* Source/WebCore/dom/LoadableSpeculationRules.h:
* Source/WebCore/layout/formattingContexts/grid/GridFormattingContext.h:
* Source/WebCore/layout/formattingContexts/grid/GridItemRect.h:
* Source/WebCore/layout/formattingContexts/grid/GridLayout.h:
* Source/WebCore/layout/formattingContexts/grid/ImplicitGrid.h:
* Source/WebCore/layout/formattingContexts/grid/TrackSizingAlgorithm.h:
* Source/WebCore/layout/formattingContexts/grid/TrackSizingFunctions.h:
* Source/WebCore/layout/formattingContexts/grid/UnplacedGridItem.h:
* Source/WebCore/layout/formattingContexts/grid/UsedTrackSizes.h:
* Source/WebCore/loader/DocumentPrefetcher.h:
* Source/WebCore/loader/cache/CachedResourceHandle.h:
* Source/WebCore/platform/graphics/AudioVideoRenderer.h:
* Source/WebCore/platform/graphics/avfoundation/AudioVideoRendererAVFObjC.h:
* Source/WebCore/style/values/backgrounds/StyleBackgroundLayer.h:
* Source/WebCore/style/values/backgrounds/StyleBackgroundSize.h:
* Source/WebCore/style/values/backgrounds/StyleFillLayers.h:
* Source/WebCore/style/values/backgrounds/StyleRepeatStyle.h:
* Source/WebCore/style/values/masking/StyleMaskLayer.h:

Canonical link: https://commits.webkit.org/301320@main
